src/EnergyPlus/UtilityRoutines.hh

```diff
@@ -220,22 +220,57 @@ void SummarizeErrors(EnergyPlusData &state);
 
 void ShowRecurringErrors(EnergyPlusData &state);
 
+
 struct ErrorObjectHeader
 {
     std::string_view routineName;
     std::string_view objectType;
     std::string_view objectName;
 };
 
-void ShowSevereItemNotFound(EnergyPlusData &state, ErrorObjectHeader const &eoh, std::string_view fieldName, std::string_view fieldValue);
-void ShowSevereInvalidKey(EnergyPlusData &state, ErrorObjectHeader const &eoh, std::string_view fieldName, std::string_view fieldValue);
-void ShowSevereEmptyField(EnergyPlusData &state,
-                          ErrorObjectHeader const &eoh,
+        // Wrappers for common errors
```

Comment:
I'm not sure I'm following this change. are they being used?

Reply:
Yes, I've been using them in my previous four or five PRs. Do a search on a search on some of these.